scripts/apps/authoring/views/article-edit.html

```diff
@@ -371,3 +371,19 @@
          data-display-user="{{ getSignOffMapping() }}"
     ></div>
 </div>
+
+<div class="field" ng-if="schema.usageterms" sd-width="{{editor.usageterms.sdWidth || 'full'}}" order="{{editor.usageterms.order}}" sd-validation-error="error.usageterms" data-required="schema.usageterms.required">
+  <label translate>Usage Terms</label>
+  <input type="text"
```

Comment:
I'm not sure if this should be input, imo in general there will be more text, so textarea would be better.